examples/3d/bloom.rs

```diff
@@ -124,40 +124,51 @@ fn update_bloom_settings(
         (entity, Some(mut bloom_settings)) => {
             *text = "BloomSettings (Toggle: Space)\n".to_string();
             text.push_str("-----------------------------\n");
-            text.push_str(&format!("(Q/W) Intensity: {}\n", bloom_settings.intensity));
-            text.push_str(&format!("(A/S) Threshold: {}\n", bloom_settings.threshold));
-            text.push_str(&format!(
-                "(D/F) Threshold Softness: {}",
-                bloom_settings.threshold_softness
-            ));
+            text.push_str(&format!("(Q/A) Intensity: {}\n", bloom_settings.intensity));
+            text.push_str(&format!("(W/S) Mid offset: {}\n", bloom_settings.mid_offset));
```

Comment:
I considered that, but decided against it because that makes it harder to test the effect and i didn't want to ruin the experience for whoever modifies it after us. On top of that, i revamped the blend function (again) and now the values make more sense. Are you sure we should change this to intensity + presets?

Reply:
Up to you on what you think is best. Leaving it as is is also fine. We can always wait and see what other people think when this gets reviewed on the main bevy repo.

Reply:
I think I'll leave it